### ISSUE TITLE
[Alternatives] Project Maintenance

### ISSUE THREAD

**Issue by yajra** [bllim/laravel4-datatables-package#215]
Hi Guys,

I am currently/actively maintaining a fork of this project. Currently supporting Laravel 4.2|5.0|5.1. If you need some alternatives, my repo is https://github.com/yajra/laravel-datatables. Most of the functionalities of this project is inherited on my fork. It would be great if we can collaborate and continue this project on my repo since this one is not being actively maintained.

Kudos to the great job done on this project. Cheers!

**Comment by awkwardusername** [comment]
I can't upvote this more than I could. Thanks @yajra.

**Comment by MarkVaughn** [comment]
I'll leave a note on the readme, I'd vote to have your changes be pulled here, but that's up to @bllim

**Comment by bgultekin** [comment]
Hi, nice job. If commits are backward compatible, that's okey to pull them here. But that's not up to me. That's up to @yajra . If he wants continue to his own repo that's ok. 

Lastly, @yajra did you forked or directly clone this project? I couldn't see anywhere on your repo forked from info. If you cloned, that's not a problem but shouldn't you mention at somewhere from origin repo?

**Comment by yajra** [comment]
@MarkVaughn @bllim, There's a lot of changes done on my repo and pulling it down here would be harder. I think I forked this project way back on Laravel 4.0 and then it was stuck there. My initial plan was to make it work on oracle database which is what I am currently using in my projects using my [Laravel-OCI8](https://github.com/yajra/laravel-oci8) package. Never have I thought that other developers  will also use it as it is today.

@bllim, I forked this using github. But when I noticed that some developers are using my version, I asked github to detached my repo to this project which makes the info gone. Btw, the credits was on [wiki](https://github.com/yajra/laravel-datatables/wiki/Credits) and I added it back to read me. Thanks!
